jcheck/src/main/java/org/openjdk/skara/jcheck/IssuesTitleCheck.java

```diff
@@ -28,10 +28,15 @@
 
 import java.util.ArrayList;
 import java.util.Iterator;
+import java.util.List;
 import java.util.logging.Logger;
+import java.util.regex.Pattern;
 
 public class IssuesTitleCheck extends CommitCheck {
     private final Logger log = Logger.getLogger("org.openjdk.skara.jcheck.issuesTitle");
+    private final static List<String> VALID_WORD_WITH_TRAILING_PERIOD = List.of("et al.", "etc.", "...");
+    private final static List<String> EXECUTABLE_NAMES = List.of("javac", "dpkg");
+    private final static Pattern FILE_OR_FUNCTION_PATTERN = Pattern.compile(".*[()/._].*");
```

Comment:
Instead of writing regexs, I think maintaining some lists here would be easier to understand.